src/data/movies.json

```diff
@@ -28,7 +28,7 @@
     "synopsis": "The early life and career of Vito Corleone in 1920s New York City is portrayed, while his son, Michael, expands and tightens his grip on the family crime syndicate.",
     "thumbnail": "https://m.media-amazon.com/images/M/MV5BMWMwMGQzZTItY2JlNC00OWZiLWIyMDctNDk2ZDQ2YjRjMWQ0XkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_QL75_UY562_CR7,0,380,562_.jpg",
     "isTrending": true,
-    "poster": "https://wallpapercave.com/wp/wp9342478.jpg"
+    "poster": "https://wallpapercave.com/wp/wp8947058.jpg"
```

Comment:
Im curious to why this was changed

Reply:
It was changed because it would fit the heroslide better